src/features/pages/p2p/index.tsx

```diff
@@ -15,7 +15,7 @@ import MainNav from 'features/components/templates/navigation/main-nav'
 import { useOpenLiveChat } from 'components/hooks/use-open-live-chat-redirection'
 import useRegion from 'components/hooks/use-region'
 import InitialLoader from 'components/elements/dot-loader'
-import PageNotFound from 'features/pages/404'
+import PageNotFound from 'features/pages/404-error'
```

Comment:
Please delete the 404-error folder as well

Reply:
You mean the old **pages/404** folder right? 
**pages/404-error** is the new 404 page.
I will remove the old folder but need to update all paths that used the old folder too